src/controls/private/QskSGNodeUtility.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <QSGNode>
+
+namespace QskSGNode
+{
+    enum AppendMode
+    {
+        Sequential,
+        Recursive
+    };
+
```

Comment:
@uwerat This function is a candidate for public API and could be moved to `QskSGNode.h` next to `ensureNode`